handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -181,7 +194,9 @@ private void sendWithRetry(Invocation invocation, AsyncResponse asyncResp,
             newExecutor = new Executor() {
                 @Override
                 public void execute(Runnable command) {
-                    command.run(); // retry的场景，对于同步调用， 需要在网络线程中进行。同步调用的主线程已经被挂起，无法再主线程中进行重试。
+                    // retry的场景，对于同步调用, 同步调用的主线程已经被挂起，无法再主线程中进行重试;
```

Comment:
great, could you please add an automated test to ensure it won't be broken by changes in the future?